### ISSUE TITLE
Invoice information not correctly formatted in payment confirmation email and recurrent payment email

### ISSUE THREAD

**Issue by madalincm** [mozilla/fxa#5134]
#### Affected versions:
- All Firefox

#### Affected Platforms:
- Windows 10 (64-bit)

#### Preconditions
- existent FxA account with no subscriptions

#### Steps to reproduce
1. Create a daily product subscription (https://accounts.stage.mozaws.net/subscriptions/products/prod_H63sY3qFAwW4Rk?plan=plan_H647jpSbC33HHp).
2. Check the Payment received email that has been sent.

#### Expected result:
- The information is correctly formatted. 

#### Actual result:
- The information is displayed as plain text, with no indentations or alignments. 

#### Notes
- This is also reproducing in recurrent payment emails.
![image](https://user-images.githubusercontent.com/15685441/80485578-58edc800-8962-11ea-896f-0cd4e3a190fd.png)

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXA-1774)
┆Issue Number: FXA-1774

**Comment by stlhood** [comment]
This is actually expected behavior. Formerly in staging (when emails were sent out of Salesforce) the results did indeed look like the "expected result" screenshot, but that had always been in error. In a nutshell, there were ongoing issues with formatting the invoice data in a table, and so we ultimately ended up altering the design to be a simple unformatted list. the "actual result" screenshot reflects that updated design.

So in summary, I recommend this issue be closed.

**Comment by clouserw** [comment]
Thanks @stlhood .  Closing

**Comment by data-sync-user** [comment]
➤ Wil Clouser commented:

won't do

**Comment by madalincm** [comment]
I was not aware of that change. Sorry for the noise. Marking the bug as invalid.
